.gitignore

```diff
@@ -0,0 +1,2 @@
+cypress/videos
```

Comment:
What about adding a local .gitignore file in cypress?

Reply:
Maybe, someday there will be more to be ignored

Reply:
Done ✅